infra/elasticsearch/main.tf

```diff
@@ -123,11 +114,12 @@ resource aws_elasticsearch_domain elasticsearch {
     automated_snapshot_start_hour = 23
   }
 
-  tags {
-    Domain = "${var.DSS_ES_DOMAIN}"
-    CreatedBy = "Terraform"
-    Application = "DSS"
-  }
+    tags = "${merge(
+        local.common_tags,
+        map(
```

Comment:
Is writing `Domain = "${var.DSS_ES_DOMAIN}"` clearer than a 3 line map?

Reply:
from what i understand through what TF has available, there is not.  its possible to break this into 1 line, but there its gonna be harder to manage the resource-specfic values

Reply:
Alright, sounds best to leave it as is then.